week1/conf/index-bbuy.logstash.conf

```diff
@@ -0,0 +1,102 @@
+            "/product/frequentlyPurchasedWith/*/text()", "frequentlyPurchasedWith",
+            "/product/accessories/*/text()", "accessories",
+            "/product/relatedProducts/*/text()", "relatedProducts",
+            "/product/crossSell/text()", "crossSell",
+            "/product/salesRankShortTerm/text()", "salesRankShortTerm",
+            "/product/salesRankMediumTerm/text()", "salesRankMediumTerm",
+            "/product/salesRankLongTerm/text()", "salesRankLongTerm",
+            "/product/bestSellingRank/text()", "bestSellingRank",
+            "/product/url/text()", "url",
+            "/product/categoryPath/*/name/text()", "categoryPath", 
+            "/product/categoryPath/*/id/text()", "categoryPathIds", 
+            "/product/categoryPath/category[last()]/id/text()", "categoryLeaf",
+            "count(/product/categoryPath/*/name)", "categoryPathCount",
+            "/product/customerReviewCount/text()", "customerReviewCount",
+            "/product/customerReviewAverage/text()", "customerReviewAverage",
+            "/product/inStoreAvailability/text()", "inStoreAvailability",
+            "/product/onlineAvailability/text()", "onlineAvailability",
+            "/product/releaseDate/text()", "releaseDate",
+            "/product/shippingCost/text()", "shippingCost",
+            "/product/shortDescription/text()", "shortDescription",
+            "/product/shortDescriptionHtml/text()", "shortDescriptionHtml",
+            "/product/class/text()", "class",
+            "/product/classId/text()", "classId",
+            "/product/subclass/text()", "subclass",
+            "/product/subclassId/text()", "subclassId",
+            "/product/department/text()", "department",
+            "/product/departmentId/text()", "departmentId",
+            "/product/bestBuyItemId/text()", "bestBuyItemId",
+            "/product/description/text()", "description",
+            "/product/manufacturer/text()", "manufacturer",
+            "/product/modelNumber/text()", "modelNumber",
+            "/product/image/text()", "image",
+            "/product/condition/text()", "condition",
+            "/product/inStorePickup/text()", "inStorePickup",
+            "/product/homeDelivery/text()", "homeDelivery",
+            "/product/quantityLimit/text()", "quantityLimit",
+            "/product/color/text()", "color",
+            "/product/depth/text()", "depth",
+            "/product/height/text()", "height",
+            "/product/weight/text()", "weight",
+            "/product/shippingWeight/text()", "shippingWeight",
+            "/product/width/text()", "width",
+            "/product/longDescription/text()", "longDescription",
+            "/product/longDescriptionHtml/text()", "longDescriptionHtml",
+            "/product/features/*/text()", "features" 
+        ]
+      }
+
+      mutate {
+         # clean up some stuff we don't need
+        remove_field => ["message", "host", "path"]
```

Comment:
I was able to add some convert to mutation and change the types from string to float
e.g.
convert => {
"regularPrice" => "float"
}